src/NodaTime.Test/Text/AnnualDateTypeConverterTest.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 The Noda Time Authors. All rights reserved.
+// Use of this source code is governed by the Apache License 2.0,
+// as found in the LICENSE.txt file.
+
+using JetBrains.Annotations;
+using NodaTime.Text;
+
+namespace NodaTime.Test.Text
+{
+    [UsedImplicitly]
+    public class AnnualDateTypeConverterTest : TypeConverterBaseTestBase<AnnualDate>
+    {
+        [NotNull] [UsedImplicitly] static readonly string[] RoundtripData =
+        {
+            AnnualDatePattern.Iso.Format(default),
```

Comment:
For these tests, I wonder whether the literal values would be clearer, and possibly rather than *just* round-tripping, provide the actual value as well.
So a test would be of the form:

```csharp
new AnnualDate(1, 1), "01-01"
```

(Or parameterized as `[TestCase(1, 1, "01-01")]`)

Otherwise if the type converter just formatted everything as "" and used a default value, the tests might pass. (I haven't double checked that.)